docs/development.md

```diff
@@ -278,31 +278,31 @@ SSH_KEY_FILE=.sshkey
 rm -f "${SSH_KEY_FILE}" 2>/dev/null
 ssh-keygen -t rsa -b 2048 -f "${SSH_KEY_FILE}" -N '' 1>/dev/null
 echo "Machine SSH key generated in ${SSH_KEY_FILE}"
-export AZURE_SSH_PUBLIC_KEY=$(cat "${SSH_KEY_FILE}.pub" | base64 | tr -d '\n')
+export AZURE_SSH_PUBLIC_KEY=$(cat "${SSH_KEY_FILE}.pub" | base64 | tr -d '\r\n')
 
 # To populate secret in azure.json file.
 export AZURE_STANDARD_JSON_B64=$(echo '{
-      "cloud": "${AZURE_ENVIRONMENT}",
-      "tenantId": "${AZURE_TENANT_ID}",
-      "subscriptionId": "${AZURE_SUBSCRIPTION_ID}",
-      "aadClientId": "${AZURE_CLIENT_ID}",
-      "aadClientSecret": "${AZURE_CLIENT_SECRET}",
-      "resourceGroup": "${CLUSTER_NAME}",
-      "securityGroupName": "${CLUSTER_NAME}-node-nsg",
-      "location": "${AZURE_LOCATION}",
-      "vmType": "standard",
-      "vnetName": "${CLUSTER_NAME}-vnet",
-      "vnetResourceGroup": "${CLUSTER_NAME}",
-      "subnetName": "${CLUSTER_NAME}-node-subnet",
-      "routeTableName": "${CLUSTER_NAME}-node-routetable",
-      "loadBalancerSku": "standard",
-      "maximumLoadBalancerRuleCount": 250,
-      "useManagedIdentityExtension": false,
-      "useInstanceMetadata": true
-}' | envsubst | base64 | tr -d '\n')
+    "cloud": "${AZURE_ENVIRONMENT}",
+    "tenantId": "${AZURE_TENANT_ID}",
+    "subscriptionId": "${AZURE_SUBSCRIPTION_ID}",
+    "aadClientId": "${AZURE_CLIENT_ID}",
+    "aadClientSecret": "${AZURE_CLIENT_SECRET}",
+    "resourceGroup": "${CLUSTER_NAME}",
+    "securityGroupName": "${CLUSTER_NAME}-node-nsg",
+    "location": "${AZURE_LOCATION}",
+    "vmType": "standard",
+    "vnetName": "${CLUSTER_NAME}-vnet",
+    "vnetResourceGroup": "${CLUSTER_NAME}",
+    "subnetName": "${CLUSTER_NAME}-node-subnet",
+    "routeTableName": "${CLUSTER_NAME}-node-routetable",
+    "loadBalancerSku": "standard",
+    "maximumLoadBalancerRuleCount": 250,
+    "useManagedIdentityExtension": false,
+    "useInstanceMetadata": true
+}' | envsubst | base64 | tr -d '\r\n')
 
 # VMSS requires different values
-export AZURE_VMSS_JSON_B64=$(echo "$AZURE_STANDARD_JSON_B64" | base64 -d | jq '.vmType = "vmss"' | base64 | tr -d '\n')
+export AZURE_VMSS_JSON_B64=$(echo "$AZURE_STANDARD_JSON_B64" | base64 -d | jq '.vmType = "vmss"' | base64 | tr -d '\r\n')
```

Comment:
I don't think we actually need both `AZURE_STANDARD_JSON_B64` and `AZURE_VMSS_JSON_B64` since we're using "vmss" vmType everywhere. We can just replace `AZURE_STANDARD_JSON_B64`with `AZURE_JSON_B64` and set the vmType to "vmss".

Reply:
SGTM. Is it possible to get this PR merged first so we can have some test signals for our testgrid jobs over the long weekend? I will address this issue in a follow-up PR.

Reply:
SGTM